<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameDemoWidget</class>
 <widget class="QWidget" name="FrameDemoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>418</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Layout direction:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>directionComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="directionComboBox">
       <item>
        <property name="text">
         <string>Left to Right</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Right to Left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Top to Bottom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bottom to Top</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="frameLayout" stretch="1,1,1">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Frame</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="raisedFrameRadioButton">
          <property name="text">
           <string>&amp;Raised</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="plainFrameRadioButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string comment="Flat frame. No frame is actually drawn.">&amp;Flat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="sunkenFrameRadioButton">
          <property name="text">
           <string>S&amp;unken</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="flatGroupBoxCheckBox">
          <property name="text">
           <string comment="Flat group box. No frame is actually drawn">Flat</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab Widget</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4"/>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="msgLayout" stretch="1">
     <item>
      <widget class="QFrame" name="msgFrame">
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>directionComboBox</tabstop>
  <tabstop>flatGroupBoxCheckBox</tabstop>
  <tabstop>raisedFrameRadioButton</tabstop>
  <tabstop>plainFrameRadioButton</tabstop>
  <tabstop>sunkenFrameRadioButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
